lib/GLPI/Agent/HTTP/Client.pm

```diff
@@ -607,7 +604,11 @@ sub _KeyChain_or_KeyStore_Export {
             command => "security find-certificate -a -p > '$file'",
             logger  => $logger
         );
-        @certs = IO::Socket::SSL::Utils::PEM_file2certs($file)
+        getAllLines(
+            command => "security find-certificate -a -p /System/Library/Keychains/SystemRootCertificates.keychain >> '$file'",
+            logger  => $logger
+        );
+        push @certs, IO::Socket::SSL::Utils::PEM_file2certs($file)
```

Comment:
That was not my idea.

I prefer to change the command on a given "ssl-keystore" value.

Here you force the run of 2 commands where this is not required for most people as Mozilla::CA should provide the public CA certificates.

I would prefer something like:
```#perl
my $command = "security find-certificate -a -p";
$command .= " /System/Library/Keychains/SystemRootCertificates.keychain"
    if $self->{ssl_keystore} =~ /^system-ssl-ca$/i;
getAllLines(
     command => "$command > '$file'",
     logger  => $logger
);
```

and even, on l.695, the test on Mozilla::CA should be changed to not load it is system public ca are used as this would be redondant:
```#perl
if (($OSNAME ne 'darwin' || $self->{ssl_keystore} !~ /^system-ssl-ca$/i) && Mozilla::CA->require()) {
```